DigitalLearningSolutions.Data.Migrations/202201060948_AddV2AppBaseUrl.cs

```diff
@@ -0,0 +1,23 @@
+namespace DigitalLearningSolutions.Data.Migrations
```

Comment:
Looking at this I have a feeling that for any version of this app that has not run this migration but already has a config by name of `V2AppBaseUrl` may end up with 2 rows with the same name given `ConfigName` does not have any constriants on it.
My DB does not have this config value so this will work fine but I think Showkaths did so running this migration would lead to 2 rows with the same name.

Reply:
cool I've changed it so it so it checks there isn't one with the same config value before it add, the down still removes anything with that configName

Reply:
This has got me thinking on whether we need a `down`. Given the original concern that the value may already exist, if that is true, then removing this value may inadvertently break anything that was reliant on this value prior to this point.

Reply:
Nice one thanks.

Reply:
This migration is not useful as it will not allow the application to work on all environments, i.e. the value should not be test on any environment other than test (and maybe dev in certain cases). I've commented about it on the place where we're accessing this value as well.

I also don't think it will create the correct url in its current setup, the old one from ConfigService.TrackingSystemBaseUrl works because the url stored in the database is https://www.dls.nhs.uk/tracking/